tests/TestGranger.py

```diff
@@ -54,8 +52,10 @@ def test_granger_test(self):
         dataset = self.syn_data_2d()
         G = Granger()
         p_value_matrix, adj_matrix = G.granger_test_2d(data=dataset)
-        print('P-value matrix is \n {}'.format(p_value_matrix))
-        print('Adjacency matrix is \n {}'.format(adj_matrix))
+        p_value_matrix_truth = np.array([[0, 0.5989, 0, 0.5397], [0.0006, 0, 0.0014, 0]])
+        adj_matrix_truth = np.array([[1, 0, 1, 0], [1, 1, 1, 1]])
```

Comment:
could you remind me how you get this truth_value?

Reply:
It was computed from the current version [1ebf232](https://github.com/cmu-phil/causal-learn/commit/1ebf232f0244a60fa7908bdea2b1bdf451da4074)

Reply:
cool, please add comments on how the p-values are generated --- same as all other PRs (it's a good practice to check other PRs before writing the PR so that you know what are the best practices we follow currently :) ) This can reduce the review overhead.

Reply:
Cool, thanks.